### PR TITLE
[strategies] [test] Fix spherical cross track and convex hull tests

### PR DESCRIPTION
**Pull request by vissarion**
This PR fixes some tests that fail after merging https://github.com/boostorg/geometry/pull/756
See also https://app.circleci.com/pipelines/github/vissarion/geometry/58/workflows/44e84067-89f7-4938-947f-006156956840

### CODE REVIEW

File: test/algorithms/Jamfile [hunk 1]
```diff
@@ -25,8 +25,6 @@ test-suite boost-geometry-algorithms
     [ run comparable_distance.cpp      : : : : algorithms_comparable_distance ]
     [ run convert.cpp                  : : : : algorithms_convert ]
     [ run convert_multi.cpp            : : : : algorithms_convert_multi ]
-    [ run convex_hull.cpp              : : : : algorithms_convex_hull ]
-    [ run convex_hull_multi.cpp        : : : : algorithms_convex_hull_multi ]
     [ run correct.cpp                  : : : : algorithms_correct ]
```
Comment (barendgehrels) [inline test/algorithms/Jamfile:28]: So it's gone? Will it be fixed later?

**Review by barendgehrels — APPROVED** [review]
Looks OK but I don't know the reason to exclude convex hull tests.

File: test/algorithms/Jamfile (hunk 1 above)
Comment (vissarion) [reply test/algorithms/Jamfile:28]: No, they are moved in a `convex_hull` directory, see https://github.com/boostorg/geometry/pull/756/commits/eae06f70dfb2f499810564a636039ec6474c10ac

Comment (barendgehrels) [reply test/algorithms/Jamfile:28]: > 
> 
> No, they are moved in a `convex_hull` directory, see [eae06f7](https://github.com/boostorg/geometry/commit/eae06f70dfb2f499810564a636039ec6474c10ac)

I remember now, the big PR.

**Pull request merged**